README.md

```diff
@@ -13,6 +13,7 @@ See [action.yml](action.yml)
 The following parameters are always required:
 
  - API_HOST - Domain name of the ApiConnect instance where discovered APIs will be sent.<br /> &nbsp; Example : `us-east.apiconnect.automation.ibm.com`
+ - PLATFORM_API_PREFIX - Platform API prefix have a default value of `platform-api` as same as for ApiConnect on Cloud. It can be changed to the env prefix
```

Comment:
....has a default......which is the same as the prefix value for ApiConnect on Cloud. It can be changed to match your systems set up if different from the default